### ISSUE TITLE
A failure occurred when execute the command "sh webgoat_developer_bootstrap.sh"

### ISSUE THREAD

**Issue by lgonline** [WebGoat/WebGoat#145]
dear
    A failure occurred when i execute the command "sh webgoat_developer_bootstrap.sh"
    Detailed information as follows:
WARNING] The POM for org.owasp.webgoat:webgoat-container:jar:7.0-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] webgoat-plugins-parent ............................. SUCCESS [01:00 min]
[INFO] common ............................................. FAILURE [  3.354 s]
[INFO] goat-hills-financial ............................... SKIPPED
[INFO] access-control-matrix .............................. SKIPPED
[INFO] back-doors ......................................... SKIPPED
[INFO] basic-authentication ............................... SKIPPED
....
[INFO] zip-bomb ........................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:04 min
[INFO] Finished at: 2015-11-02T16:04:54+08:00
[INFO] Final Memory: 17M/173M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project common: Could not resolve dependencies for project org.owasp.webgoat.lesson:common:jar:1.0: Could not find artifact org.owasp.webgoat:webgoat-container:jar:7.0-SNAPSHOT -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :common
cp: cannot stat `./WebGoat-Lessons/target/plugins/*.jar': No such file or directory

**Comment by nbaars** [comment]
@lgonline the first build step failed (webgoat-container). The lessons are dependent on this jar file. Can you please look at the first step into the buildjob to see why the build step of the container failed?

Thanks

**Comment by lgonline** [comment]
dear nbaars, sorry for response it so late, i cannot find the webgoat-container:jar in org.owasp.webgoat.lesson:common

**Comment by d-subrahmanyam** [comment]
@lgonline - not sure if you found the issue. Ideally in the pom of the Webgoat-lessons its looking for webgoat-container-7.0-SNAPSHOT.jar while the webgoat root we are building is installing a webgoat-container-7.1-SNAPSHOT.jar. I changed the pom in webgoat-lessons module and it voila it worked.

**Comment by span** [comment]
I tried running the script on my machine and it worked without errors. I'm also looking at the POM which still has the 7.0-SNAPSHOT.

```
<groupId>org.owasp.webgoat</groupId>
<artifactId>webgoat-container</artifactId>
<version>7.0-SNAPSHOT</version>
```

@d-subrahmanyam Could you try to recreate the issue and post the complete error back here?

**Comment by rusboy** [comment]
I had absolutely same issue before. By the way, it is here https://github.com/WebGoat/WebGoat-Lessons/issues/55

@span try to clear you you local maven storage and repeat test with fresh cloned source code.. 

@lgonline 
webgoat project's branches don't provide artifact with version 7.0-SNAPSHOT. looks like it was, before tag 7.0.1 had been released.. but doesn't exist now

So, to compile lessons, you should change manually pom.xml of the webgoat-lessions. This block of the file should be inpacted:

            <dependency>
                <groupId>org.owasp.webgoat</groupId>
                <artifactId>webgoat-container</artifactId>
                <version>7.0-SNAPSHOT</version>
                <type>jar</type>
                <scope>provided</scope>
            </dependency>

You you use master or develop of webgoat project, change 7.0-SNAPSHOT to 7.1-SNAPSHOT.
If tag 7.0.1 then you should use version 7.0.1 instead.

@nbaars 
may be you release tag 7.0.1 for webgoat lessons using 7.0.1 webgoat-container as dependency.
Then, enough to set 7.1-SNAPSHOT of webgoat-container as dependency for webgoat-lessions in master branch.

**Comment by nbaars** [comment]
Close as @span fixed this with WebGoat/WebGoat-Lessons#60
